chat/src/main/java/io/skygear/plugins/chat/CacheController.java

```diff
@@ -286,4 +287,14 @@ void markPendingMessageOperationsAsFailed() {
     }
 
     //endregion
+
+    //region Chat Users
+    void didParticipants(Collection<Participant> participants) {
+        this.store.setParticipants(participants);
+    }
+
+    void getParticipants(Collection<String> participantIds, GetParticipantsCallback callback) {
```

Comment:
This is not a must, but in my opinion the api will be looks better in the cacheController set and get api are symmetric. Currently set is a collection of participants, but the in the cache get callback is a map. Converting the result from list to map better to be happen in `ChatContainer` instead of store. Now we did it twice in both `ChatContainer` and `store`.